src/SeedAddress.spec.ts

```diff
@@ -15,7 +15,12 @@ describe("SeedAddress", () => {
         port: 8776,
       },
     });
-    cy.get("span.seed-icon").should("have.text", "🌱");
+    cy.get("span.seed-icon img").should("have.attr", "alt", "🌱");
+    cy.get("span.seed-icon img").should(
+      "have.attr",
+      "src",
+      "https://twemoji.maxcdn.com/v/14.0.2/72x72/1f331.png",
```

Comment:
We should bundle the twemoji assets and serve them ourselves. Have a look at 
  - https://github.com/radicle-dev/radicle-upstream/blob/main/scripts/install-twemoji-assets.sh
  - https://github.com/radicle-dev/radicle-upstream/blob/main/package.json#L35
for inspiration.

Reply:
hey, yeah thats nice! Did remember that you did something like this on Upstream, this would allow us to not depend on the tweemoji cdn 👍

Reply:
7740f4519b138ed04d855e36abcf2cfea547e8c5